### DISCUSSION TITLE
SwiftBar menu item icon

### DISCUSSION THREAD

**Discussion by defjaf** [swiftbar/SwiftBar#377] [Q&A]
Hi,

I run swiftbar on two machines. On one, there is an icon shown next to the "SwiftBar   >" menu item:
<img width="234" alt="Screenshot 2023-05-09 at 12 20 49" src="https://github.com/swiftbar/SwiftBar/assets/799252/28e66d1c-2ec0-4429-acb1-c0b5a41e927b">

But the icon is not present on the other machine (the menu item is there, just without the icon):
<img width="283" alt="Screenshot 2023-05-10 at 10 32 55" src="https://github.com/swiftbar/SwiftBar/assets/799252/52bbb2d2-db77-48c7-8718-5796d530efdb">

I vaguely remember seeing instructions on how to change between the two behaviours, but I can't find it documented anywhere. Can someone give me a pointer?

**Comment by melonamin** [comment]
Hi @defjaf,

Yes, it was a GUI setting at some point. 

Try running this on both machines `defaults read com.ameba.SwiftBar HideSwiftBarIcon`  And switch to your liking

**Comment by defjaf** [comment]
... in fact I've had this set to 1 (true) on both of my machines for a while, but it doesn't seem to have an effect on the machine that is still showing the icon... (I tried various combinations of quitting, running `defaults delete com.ameba.SwiftBar HideSwiftBarIcon` before `defaults write ...`.)

**Reply by melonamin** [reply]
Interesting. Let me look into it, probably on the weekend.

**Reply by melonamin** [reply]
I'm returning this setting back to UI in the next version, stay tuned
